addons/email_confirmation/controllers.py

```diff
@@ -56,6 +56,33 @@ def web_auth_signup(self, *args, **kw):
 
         return request.render('auth_signup.signup', qcontext)
 
+    @http.route('/web/reset_password', type='http', auth='public', website=True)
+    def web_auth_reset_password(self, *args, **kw):
+        """Need to override, as all messages were originally in English."""
```

Comment:
Those English messages were marked for translation. Why don’t we just use gettext to translate them or change the translations? There are probably many other strings in Odoo which are originally in English and overriding them this way doesn’t look like a practical general solution…